configs/stage/permissions/content-sources.json

```diff
@@ -0,0 +1,15 @@
+{
+    "policies": [
```

Comment:
actually, this should likely be 'repositories' or 'content-sources'?

Reply:
oh derp derp derp hard...

Reply:
Shhh.... you saw nothing Justin Snow...